symfony/src/EventListener/RequestListener.php

```diff
@@ -73,7 +73,7 @@ private function checkPageNotices()
 
     private function checkSiteSelect()
     {
-        if (!$this->session->has('site')) {
+        if (!$this->session->has('site') && !preg_match('/^\/s\/cron\/.*/', $this->request->getPathInfo())) {
```

Comment:
Can we make the `/s` prefix optional like in the regex above for the admin URL pattern regex?